setup.py

```diff
@@ -27,7 +27,7 @@
 
 setup(
     author="The Allen Institute for Artificial Intelligence",
-    author_email="jeremym@allenai.org",
+    author_email="oliver.elbert@noaa.gov",
```

Comment:
Do we need to change any more of the information in the setup block such as `python-requires` and some of the other classifiers?

Reply:
Updated the `python-requires` will wait for opinions of others to make subsequent changes.

Reply:
If we don't expect to release the code on pypi, this has minimal impact since most of that metadata is made for pypi and other tools that automate gathering informations on the packages.

That said, no reason to not change the other field to shift things to NOAA, correct license and Py 3.11 as a base py

Reply:
If we're going to change this (we should) we should change it everywhere so there is no inconsistency